sql-plugin/src/main/scala/org/apache/spark/sql/rapids/complexTypeExtractors.scala

```diff
@@ -17,6 +17,7 @@
 package org.apache.spark.sql.rapids
 
 import ai.rapids.cudf.{ColumnVector, Scalar}
+import com.nvidia.spark.RebaseHelper.withResource
```

Comment:
I do not know this `withResource`, but we had better use this the one from `Arm.scala` by extending your class/object from `Arm`.

Reply:
Nice catch.